src/components/activity-introduction/introduction-page-content.tsx

```diff
@@ -1,17 +1,27 @@
-import React from "react";
+import React, { useEffect, useState } from "react";
 import { ActivitySummary } from "../activity-introduction/activity-summary";
 import { ActivityPageLinks } from "../activity-introduction/activity-page-links";
-import { Activity } from "../../types";
+import { Activity, TeacherFeedback } from "../../types";
+import { watchActivityLevelFeedback } from "../../firebase-db";
+import { ActivityLevelFeedback } from "./activity-level-feedback";
 
 import "./introduction-page-content.scss";
 
 interface IProps {
   activity: Activity;
+  isInSequence?: boolean;
   onPageChange: (page: number) => void;
 }
 
 export const IntroductionPageContent: React.FC<IProps> = (props) => {
-  const { activity, onPageChange } = props;
+  const { activity, onPageChange, isInSequence } = props;
+  const [feedback, setFeedback] = useState<TeacherFeedback | null>(null);
+
+  useEffect(() => {
+    if (activity.id) {
+      return watchActivityLevelFeedback(fb => setFeedback(fb));
```

Comment:
can we return a function here to stop watching activity level feedback when the page unmounts?